cfgov/v1/util/ref.py

```diff
@@ -7,21 +7,21 @@
     ('document', 'Report'),
     ('date', 'Events'),
     ('microphone', 'Speech'),
-    ('bullhorn', 'Press Release'),
-    ('contract', 'Op-Ed'),
+    ('bullhorn', 'Press release'),
+    ('contract', 'Op-ed'),
     ('double-quote', 'Testimony'),
 ]
 
 related_posts_categories = [
     ('Blog', (
         ('At the CFPB', 'At the CFPB'),
-        ('Policy &amp; Compliance', 'Policy & Compliance'),
-        ('Data, Research &amp; Reports', 'Data, research & reports'),
+        ('Policy &amp; Compliance', 'Policy and compliance'),
+        ('Data, Research &amp; Reports', 'Data, research, and reports'),
         ('Info for Consumers', 'Info for consumers'),
```

Comment:
The first string in these tuples (the three lines above) should match the second string, right?

Reply:
D'oh, sorry. Didn't see your comment while I was reviewing :)